infrasim/cloud_img.py

```diff
@@ -0,0 +1,88 @@
+'''
+*********************************************************
+Copyright @ 2015 EMC Corporation All Rights Reserved
+*********************************************************
+'''
+
+import os
+import string
+import random
+from infrasim import run_command
+
+
+# def load_backing_file we can add load backing file from cloud image offical website later
+
+def gen_qemuimg(boot_img):
+    status, output = run_command("qemu-img create -f qcow2 -o backing_file=/home/infrasim/jenkins/data/"
```

Comment:
fixed path is not safe for user. ☠️ This is a package instead of a test case.

Reply:
I use this temp path for test purpose only, the folder will be rm after test